### PR TITLE
Fix Selecting "All Threads" and Display That Track on Linux

### PR DESCRIPTION
**Pull request by florian-kuebler**
With recent changes, we started using "-1" instead of "0" for special
purpose thread/process ids (no selection and all threads). By these
changes, we missed a couple of places.

This change uses SamplingProfiler::kAllThreadsFakeTid on all these
places and thus fixes the issue that scheduler track was not highlighting
anything, when clicking on the "all threads" track and also displays that
track on Linux at all.

Bug: http://167391339 http://167393367
Test: In capture, select different tracks (including all threads), and
      click on the background (to reset). Also check the lable in
      TopDownView and the SamplingViews.

### CODE REVIEW

**Review by IrinaShkviro — APPROVED** [review]

**Comment by florian-kuebler** [comment]
As this is targeting the release branch, I will wait for a second approval here.

File: OrbitCore/EventBuffer.cpp
Comment (dpallotti) [inline OrbitCore/EventBuffer.cpp:45]: Maybe rename this variable, as the 0 was indeed referring to the "fake" thread id.

**Review by dpallotti — APPROVED** [review]
Thank you!

File: OrbitCore/EventBuffer.cpp
Comment (florian-kuebler) [reply OrbitCore/EventBuffer.cpp]: Done. Thanks!

File: OrbitCore/EventBuffer.cpp
Comment (dimitry-) [inline OrbitCore/EventBuffer.cpp:22]: I think we should rename it to just kAllThreadsId, it is technically not real but this is implied anyways. (not for this PR)

**Review by dimitry- — APPROVED** [review]

File: OrbitCore/EventBuffer.cpp
Comment (florian-kuebler) [reply OrbitCore/EventBuffer.cpp:22]: I'll do so! Thanks!

**Pull request merged**
